policyengine_us/parameters/gov/states/la/tax/income/exemptions/personal/dependent.yaml

```diff
@@ -0,0 +1,16 @@
+description: Louisiana provides an exemption of this amount for each dependent. 
+values:
+  # Tax Foundation confirms it's been $1,000 since at least 1983.
```

Comment:
add the source for this

Reply:
all yaml

Reply:
what kind of source?

Reply:
the tax foundation source that is mentioned

Reply:
i think it is just the first reference.